focal/4_gs1_identification.html

```diff
@@ -0,0 +1,227 @@
+            Prefix license credential to prove that it has the right to issue the key to SA&amp;E. To complete the
+            validation, SA&amp;E requires the GS1 Prefix license credential issued to GS1 Utopia, which is publicly
+            accessible and discoverable via the GS1 Company Prefix license credential.
+      </p>
+      <h4>Trust Hierarchy</h4>
+      <ul>
+            <li>
+                  GS1 Global Office is responsible for management of the GS1 identification system as a whole. It is
+                  liable for ensuring that the GS1 Prefix licenses that it issues are unique.
+            </li>
+            <li>
+                  GS1 Utopia is responsible for management of the GS1 identification system within the range(s) of the
+                  GS1 Prefix(es) issued to it. It is liable for ensuring that the GS1 Company Prefix licenses that it
+                  issues are unique.
+            </li>
+            <li>
+                  Healthy Tots is responsible for management of the GS1 identification system within the range(s) of the
+                  GS1 Company Prefix(es) issued to it. It is liable for ensuring that the GS1 identification keys that
+                  it issues are unique.
+            </li>
+            <li>
+                  SA&amp;E is responsible for ensuring that no two products listed on its website carry the same GTIN.
+            </li>
+      </ul>
+      <h4>Variation - License Transfer</h4>
+      <p>
+            GS1 license Verifiable Credentials are issued with a <code>validFrom</code> property but not a <code>validUntil</code> property. Licenses are renewable as long as the licensee abides by the terms and conditions of the GS1 Member Organization that issued the license, including regular license payment if required. Accordingly, the only way for a trading partner to know that a license is no longer valid is to check its status for revokation.
+      </p>
+      <p>
+            Normally, GS1 license Verifiable Credentials are non-transferrable, but the whole or partial acquisition of a company may require it. Suppose that Healthy Tots is acquired by Benevolent Conglomerate. Benevolent Conglomerate may decide on a hands-off approach and leave Healthy Tots to continue its operations much as before, with no impact on the way that the GS1 identification keys are managed. It's possible, though, that Benevolent Conglomerate will decide to discontinue Healthy Tots as a separate entity and instead absorb its products into a central catalogue. In this case, the GS1 Company Prefix license, originally issued to Healthy Tots, will be transferred by GS1 Utopia to Benvolent Conglomerate.
+      </p>
+      <p>
+            In this case, a status check of the original GS1 Company Prefix license Verifiable Credential should indicate a status of "transferred" and the ID of the replacement. The maintain continuity of supply chain management, the following must be supported:
```

Comment:
We need to discuss the transfer use case in more detail. Revoking the original company prefix credentials could be problematic. When a credential is revoked there is no way to return a status that it has been transferred  we can only check that it has been revoked.

Reply:
Added to discussion topics for internal meeting, will return to the PR with changes.